src/test/coll.rs

```diff
@@ -467,75 +467,32 @@ async fn empty_insert() {
     };
 }
 
-fn get_allow_disk_use(event_client: &EventClient) -> bool {
-    event_client
-        .command_events
-        .read()
-        .unwrap()
-        .iter()
-        .any(|event| match event {
-            CommandEvent::CommandStartedEvent(CommandStartedEvent {
-                command_name,
-                command,
-                ..
-            }) => {
-                if command_name == "find" {
-                    match command.get_bool("allowDiskUse") {
-                        Ok(b) => b,
-                        _ => false,
-                    }
-                } else {
-                    false
-                }
-            }
-            _ => false,
-        })
-}
-
 #[cfg_attr(feature = "tokio-runtime", tokio::test)]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 #[function_name::named]
 async fn find_allow_disk_use() {
```

Comment:
Since these test functions aren't using the `function_name!()` macro anymore, the `#[function_name::named]` annotation can be removed from each of them.